framework/src/modules/token/cc_commands/cc_transfer.ts

```diff
@@ -56,99 +53,109 @@ export class CCTransferCommand extends BaseCCCommand {
 
 	public init(args: { minBalances: MinBalance[]; supportedTokenIDs: Buffer[] }): void {
 		this._supportedTokenIDs = args.supportedTokenIDs;
-		this._minBalances = args.minBalances;
+		// this._minBalances = args.minBalances;
 	}
 
-	public async execute(ctx: CCCommandExecuteContext): Promise<void> {
-		const { ccm } = ctx;
+	public async verify(ctx: CCCommandExecuteContext): Promise<void> {
+		const { ccm, ccmSize } = ctx;
 		const methodContext = ctx.getMethodContext();
 		const { id: ownChainID } = await this._interopMethod.getOwnChainAccount(methodContext);
 		let params: CCTransferMessageParams;
-		let tokenChainID;
-		let tokenLocalID;
+		let tokenChainID: Buffer;
+		const { sendingChainID, status, fee } = ccm;
+
 		try {
```

Comment:
I think with the new definition, this try-catch is not needed?

Reply:
I think we need it to verify the params?